docs/guides/modules/integration/pages/github-apps-integration.adoc

```diff
@@ -121,20 +117,17 @@ A subset of built-in environment variables are not available in GitHub-based pro
-
-* xref:orchestrate:custom-webhooks.adoc[Custom webhooks]
-* link:https://discuss.circleci.com/t/circleci-config-suggestions-bot/47918[CircleCI's config suggestions bot]
-* The ability to use xref:orchestrate:set-up-multiple-configuration-files-for-a-project.adoc[multiple configuration files within one project]
+Before attempting to move your information from an org integrated with the GitHub OAuth app to an org integrated with CircleCI’s GitHub App, consider the following:
 
-You can not currently automate migrating your organization from the GitHub OAuth app to CircleCI's GitHub App integration.
+* If the motivation for moving is to leverage new functionality that is only available to the GitHub App integration, consider using your existing organization and installing the GitHub App alongside your OAuth app integration, as described in xref:integration:using-the-circleci-github-app-in-an-oauth-org.adoc[this guide]. 
+* If the motivation is to completely remove the OAuth app integration, please follow the instructionwe recommend first confirming that you do not immediately need any of the functionality listed in the <<currently-not-supported>> section below. If you have a dedicated account team at CircleCI, contact them first to discuss your migration.
 
-Before attempting to move your information from an org integrated with the GitHub OAuth app to an org integrated with CircleCI's GitHub App, consider the following:
+[#Steps-to-migrate-to-an-organization-without-default-GitHub-OAuth-integration
+=== Steps to migrate to an organization without default GitHub OAuth integration
 
-* If the motivation for moving is to leverage new functionality that is only available to the GitHub App integration, consider using your existing organization and installing the GitHub App alongside your OAuth app integration as described in our link:https://discuss.circleci.com/t/product-update-using-github-app-functionality-in-a-github-oauth-app-organization/52204[community forum].
-* If the motivation is to completely remove the OAuth app, confirm that you do not immediately need any of the functionality listed in the <<currently-not-supported>> section below.  If you have a dedicated account team at CircleCI, contact them first to discuss your migration.
+The following steps will guide you through migrating from an organization integrated with GitHub through the OAuth integration by default (identifyable by `/github/` or `/gh/` in the URL) to an organization that does not have a default OAuth integration with GitHub (identifyable by `/circleci/` in the URL).
 
-If you would like to switch from the OAuth app integration to the GitHub App integration, follow these steps:
+CAUTION: You can not currently automate migrating your organization from the GitHub OAuth app to CircleCI's GitHub App integration.
```

Comment:
@rosieyohannan how do I make this and the following "Caution" be grouped together?

Reply:
(found out how)